.gitignore

```diff
@@ -1,3 +1,4 @@
 lib
 dist
 node_modules
+awskeys.sh
```

Comment:
Take a look at your own PRs before merging. (A habit that I've also gotten out of lately!)